app/views/sentences/search.ctp

```diff
@@ -102,6 +103,20 @@ if ($search_disabled) {
         );
     ?></p>
 <?
+  } else {
+?>
+    <h2><?php echo __('Search error'); ?></h2>
+    <p><?php
+        echo format(
+            __(
+                'Invalid query. '.
+                'Please refer to '.
+                '<a href="{}">text search wiki</a> for more details.', true),
```

Comment:
`Please refer to text search wiki...` sounds a bit weird.
Maybe `Please refer to the search documentation...` instead.